package.json

```diff
@@ -63,14 +63,14 @@
     "formik": "^2.2.6",
     "gh-pages": "^2.2.0",
     "microbundle-crl": "^0.13.10",
-    "node-sass": "^4.0.0",
+    "sass": "^1.44.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.0.4",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "^3.4.1",
-    "typescript": "^4.2.4"
+    "typescript": "3.9.9"
```

Comment:
why is this being downgraded?

Reply:
npm 8.1.0 has stricter peer dependency rules and react-scripts 3.* expects >= typescript to be <= 3.*.  If we want to update react-scripts to 4.* that fixes the Typescript peer dependency issue.

Reply:
Hmm I wonder why the tests fail. Would it be much to update those?

Reply:
I spent some time looking at what the tests would take to update, but it looks like that would be quite a rabbit hole to go down.  Specifically, tests fail in one file (`FormCard.test.js`) and all of them appear to be related to the `jest.spyOn` method.  Here is an example of a failing test:
<img width="1032" alt="image" src="https://user-images.githubusercontent.com/32783091/145257106-1b0e226b-fdc1-44c7-ab04-4c6f89cb601a.png">